node-tests/fixtures/initializer-test/module-unification/default.js

```diff
@@ -0,0 +1,26 @@
+import Application from '@ember/application';
+import { run } from '@ember/runloop';
+
+import { initialize } from 'my-app/init/initializers/foo';
```

Comment:
the addition of `init/` is the only new thing in these fixture files. 

There's a possibility that these can get out of date if changes are done outside of MU. While not ideal the risk seems minimum given how infrequent these type of changes are and duplicating these will result in easier clean-up and is akin to what was done in [`ember-cli`](https://github.com/ember-cli/ember-cli/tree/master/tests/fixtures/module-unification-app)